tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -174,6 +155,18 @@ def dispatch_notification(self, alert):
         server = None
         try:
             self.set_destinations()
+            if (
```

Comment:
are you saying like i have to move set_destination() logic to get_alert_destinations()?